### PR TITLE
add master page user

### PR DESCRIPTION
**Pull request by ghost**
## Description
add master page user
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: resources/views/user/layout/footer.blade.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<footer>
```
Comment (at-dungle) [inline resources/views/user/layout/footer.blade.php]: chỉnh space

File: resources/views/user/layout/footer.blade.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	<div class="footer-coppyright">
+		<div class="container">
+			<div class="row">
+				<div class="col-sm-6 col-xs-12 coppyright">{{ __('home.user.footer.copyright')}}<a href="#">{{ __('home.user.footer.food_drink') }}</a>{{ __('home.user.footer.all_right') }}</div>
+			</div>
+		</div>
+	</div>
+</footer>
+<a href="#" class="totop"> </a>
```
Comment (at-dungle) [inline resources/views/user/layout/footer.blade.php]: đặt tên class v là sao?

File: resources/views/user/layout/header.blade.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<!-- Header -->
+<header>
+	<div class="header-container">
```
Comment (at-dungle) [inline resources/views/user/layout/header.blade.php]: chỉnh lại space

File: resources/views/user/layout/nav.blade.php [hunk 4]
```diff
@@ -0,0 +1,148 @@
+<nav>
+	<div class="container">
```
Comment (at-dungle) [inline resources/views/user/layout/nav.blade.php]: chỉnh hết lại space

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**